### ISSUE TITLE
Collapsed Terminal panel icon too similar to expand chevron

### ISSUE THREAD

**Issue by gregvanl** [microsoft/vscode#99204]
Testing #98994

![image](https://user-images.githubusercontent.com/12818376/83582607-cf985b00-a4f6-11ea-8495-d13e6a3deeac.png)

It is also confusing that it appears that clicking the icon expands the panel. It is actually that if the user clicks anywhere under the expand chevron (icon or empty space), the panel will expand.

**Comment by sbatten** [comment]
@misolori thoughts? the terminal icon is similar to the chevron

**Comment by misolori** [comment]
We could maybe modify the icon a bit to add a `_` next to it but i'm pretty sure we already explored that and @Tyriar wasn't a fan.

**Comment by Tyriar** [comment]
Yes `v` seems pretty overloaded. These are what the icons can mean currently:

- `>` collapsed, click to expand (lists, views)
- `v`
  - expanded, click to collapse (lists, views)
  - maximized, click to restore (panel)
  - close/hide (notifications
- `^`
  - maximize (panel)
- `x`
  - hide, non-destructive (panel, find)
  - close, destructive (editor tabs)
  - dismiss, kind of destructive (search)

At the same time we use `-`, `☐`, (multiple boxes) and `x` in the title bar to minimize, maximize, restore and close (destructive) the _window_. I don't think this is a major issue, but if we do something about it we should try make meanings consistent everywhere.

An additional challenge I think is that trying to use `☐` for maximize in the _panel_ imo doesn't illustrate to me what it's going to do. I drew up some ideas of boxes/lines with arrows in them to better describe the panel's maximize/restore icons before with @misolori, I think that's as far as the discussion went though.

**Comment by misolori** [comment]
@Tyriar I think the discussion here is around the terminal icon when a view is collapsed in the panel. The chevron is a separate issues as it's used in a lot of places (as you outlined).

**Comment by Tyriar** [comment]
Oh, that icon 🤦. My only suggestion is using _ as well (representing prompt and cursor) similar to WT:

![image](https://user-images.githubusercontent.com/2193314/83667483-5430bb00-a583-11ea-9de7-73459af5f915.png)

**Comment by gregvanl** [comment]
@misolori When the nested panel is collapsed, clicking anywhere in the collapsed column will expand the panel. I initially thought clicking the icon was the same as clicking the chevron until I realized it was anywhere in the column.
I entered an issue that clicking a second time doesn't collapse so you don't get the toggle behavior, which might be expected. See https://github.com/microsoft/vscode/issues/99205
